internal/config/watcher_test.go

```diff
@@ -36,10 +36,11 @@ output:
 	rdr := newDummyReader(confFilePath, nil)
 
 	changeChan := make(chan struct{})
+	once := sync.Once{}
 	var updatedConf stream.Config
 	require.NoError(t, rdr.SubscribeConfigChanges(func(conf *Type) error {
 		updatedConf = conf.Config
-		close(changeChan)
+		once.Do(func() { close(changeChan) })
```

Comment:
Not sure why this handler got executed more than once on my M1... I can't reproduce it on demand